pkg/controller/controller.go

```diff
@@ -95,6 +95,13 @@ func (c *Controller) ProcessNextItem() bool {
 		c.queue.Forget(key)
 		return true
 	}
+	if metav1.HasAnnotation(pvc.ObjectMeta, AnnImportPod) {
```

Comment:
Would you mind consolidating all of these  blocks 
```
		c.queue.Forget(key)
		return true
```
to:
```
    return forgetKey(<optional-log-mgs>)
```
where _forgetKey_ looks something like:
```
var forgetkey func(msg string) bool 
     if msg != "" {
        glog.Info(msg)
     }
     c.queue.Forget(key)
     return true
}
```